browser/css/btns.css

```diff
@@ -64,12 +64,13 @@ button:not(.ui-tab):not(.ui-corner-all):not(.button-primary):not(.unobutton) {
 
 .ui-pushbutton[disabled] {
 	color: var(--color-text-lighter) !important;
-	background-color: white !important;
+	background-color: var(--color-background-lighter) !important;
+	border: 1px solid var(--color-border-lighter) !important;
 }
 
 .ui-pushbutton[disabled]:hover {
 	cursor: not-allowed !important;
-	background: white !important;
+	background: var(--color-background-darker) !important;
```

Comment:
Follow up (not a blocker for this PR) the background here( in .ui-pushbutton[disabled]:hover) is not bieng applied  due to rules from `.button-secondary:hover, button:not(.ui-corner-all):not(.button-primary):not(.unobutton):hover`

Reply:
To be honest, I don't know that much about the css rules at device-desktop.css so I can't change there that much (without destroy something).

What I don't understood is, that there are so much rules with `!important` elements.